python-client/giskard/commands/cli_server.py

```diff
@@ -96,6 +96,30 @@ def get_container(version=None, quit_if_not_exists=True) -> Optional[Container]:
             return None
 
 
+def _backend_ready(endpoint) -> bool:
+    try:
+        response = requests.get(endpoint)
+
+        if response.status_code != 200:
+            return False
+
+        return "UP" == response.json()['components']['readinessState']['status']
```

Comment:
I'm testing on dev setup, but my json structure isn't the same, I don't have `components` `readinessState`

<img width="445" alt="image" src="https://github.com/Giskard-AI/giskard/assets/7889985/21235745-fe4e-4d5f-9d5d-e19b9004df5b">

Reply:
I got a different format, will use status instead to be sure.
 
<img width="655" alt="image" src="https://github.com/Giskard-AI/giskard/assets/114553769/a2822a8d-ab02-41ca-86f9-92862ca67ce3">